package.json

```diff
@@ -12,15 +14,23 @@
     "prop-types": "^15.7.2",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-scripts": "3.3.1",
-    "typescript": "^3.7.5"
+    "react-scripts": "3.3.1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/evscott/hackapp-client.git"
```

Comment:
I also noticed that create react app removed some of the typical package.json items, so I've put them back in!